x/programs/rust/examples/counter/src/lib.rs

```diff
@@ -1,4 +1,6 @@
-use wasmlanche_sdk::{public, state_keys, types::Address, Context, Gas, Program};
+#[cfg(not(feature = "bindings"))]
+use wasmlanche_sdk::Context;
```

Comment:
Also annoying that we have to do this to avoid warning when compiling with the `bindings` feature enabled (the type isn't used in the bindings code). That could be a good argument for using a trait. 

We shall see...